src/ast/passes/semantic_analyser.cpp

```diff
@@ -408,7 +408,8 @@ void SemanticAnalyser::visit(Builtin &builtin)
   }
   else if (builtin.ident == "probe") {
     builtin.type = CreateProbe();
-    probe_->need_expansion = true;
+    if (has_wildcard(probe_->name()))
+      probe_->need_expansion = true;
```

Comment:
This doesn't feel like a sufficient solution. Compare:
```
(master) # bpftrace -e 'kprobe:vfs_read,kprobe:vfs_open { print(probe); exit(); }'
Attaching 2 probes...
kprobe:vfs_open

(probe-builtin-expansion) # bpftrace -e 'kprobe:vfs_read,kprobe:vfs_open { print(probe); exit(); }'
WARNING: Symbols not found: [ kprobe:vfs_read ]
Attaching 2 probes...
kprobe:vfs_read,kprobe:vfs_open
```
(this problem is also reflected by the codegen test change)